scripts/search_index.py

```diff
@@ -203,6 +205,9 @@ def index_routes():
     """Index the routes."""
     out = {}
     for route in routes:
+        # We do not want to index the docs for Chakra based components.
+        if route.path.startswith("/docs/library/chakra/"):
```

Comment:
Maybe we should still index them, but add a special tag to them in the document. Then just deprioritize that tag when we're actually doing the query